### PR TITLE
docs (use): add get started section

### PR DESCRIPTION
**Pull request by danburck**
# Description

This PR adds content for users to get started with Evmos. It aims to be understandable by non-developers.

### CODE REVIEW

**Comment by danburck** [comment]
@masterfung There are conflicts with `main` because I decided to create a one-pager to get started instead of a whole folder. Can we keep it like this?

File: docs/develop/index.md [hunk 1]
```diff
@@ -3,7 +3,73 @@ sidebar_position: 0
...
+# Introduction
+
+Evmos is a Cosmos-based chain with full Ethereum Virtual Machine (EVM) support. Because of this [architecture](./../protocol), tokens and assets in the network may come from different independent sources.
+
+Evmos is a scalable, high-throughput Proof-of-Stake blockchain that is fully compatible and
+interoperable with Ethereum. It's built using the [Cosmos SDK](https://github.com/cosmos/cosmos-sdk/) which runs on top of [Tendermint Core](https://github.com/tendermint/tendermint) consensus engine.
+
+Evmos allows for running vanilla Ethereum as a [Cosmos](https://cosmos.network/)
+application-specific blockchain. This allows developers to have all the desired features of
+Ethereum, while at the same time, benefit from Tendermint’s PoS implementation. Also, because it is
```
Comment (masterfung) [inline docs/develop/index.md:15]: I would spell out PoS here

File: docs/develop/index.md [hunk 2]
```diff
@@ -3,7 +3,73 @@ sidebar_position: 0
...
 
-An overview of the available documentation for developers.
+# Introduction
+
+Evmos is a Cosmos-based chain with full Ethereum Virtual Machine (EVM) support. Because of this [architecture](./../protocol), tokens and assets in the network may come from different independent sources.
+
+Evmos is a scalable, high-throughput Proof-of-Stake blockchain that is fully compatible and
+interoperable with Ethereum. It's built using the [Cosmos SDK](https://github.com/cosmos/cosmos-sdk/) which runs on top of [Tendermint Core](https://github.com/tendermint/tendermint) consensus engine.
+
+Evmos allows for running vanilla Ethereum as a [Cosmos](https://cosmos.network/)
```
Comment (masterfung) [inline docs/develop/index.md:13]: not sure about the use of `vanilla` here to describe Ethereum

**Review by masterfung — CHANGES_REQUESTED** [review]
a few small changes requested.

**Review by masterfung — APPROVED** [review]

**Pull request merged**